examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,51 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+1. Create a new App at https://developers.facebook.com/apps
+2. Add the Audience Network product to it
+3. Create a new Placement to generate your `placementId`
+4. To test, ensure the `User-Agent` request header represents a mobile device
+
```

Comment:
Will the FB UI indicate that a `placementId` needs to be enabled for header bidding in the future? If not, perhaps a comment on how to action the `"Application not authorised for header bidding"` response message.

Good addition however.